src/hammer-vlsi/hammer_vlsi/vendor/openroad.py

```diff
@@ -40,13 +41,16 @@ def env_vars(self) -> Dict[str, str]:
         return reduce(add_dicts, [dict(super().env_vars)] + list_of_vars, {})
 
     def validate_openroad_installation(self) -> None:
+        # TODO: why does OPENROAD need to be an env var??
```

Comment:
OpenROAD doesn't seem to have an install convention for its tools, so I wasn't sure what purpose an `OPENROAD` environment variable would serve. For `CADENCE_HOME` it makes more sense, because the default binary path for various tools is set based on this variable + the tool version.

Reply:
Oh, I also missed the fact that you're not checking the `OPENROAD` env var, just the `openroad` and `klayout` executables. Nevermind, except remove excess code.